src/components/Header/index.js

```diff
@@ -20,19 +20,24 @@ const Header = () => (
         </label>
         <input id="search-field" type="text" placeholder="Товар, услуга" />
       </div>
-      <button type="button" className="btn--location">
-        <img className="img--marker" src={locationHeader} alt="Location" />
-        <p>Вся Беларусь</p>
-      </button>
+      <Button
+        className="btn--location"
+        img={{src: locationHeader, className: 'img--marker', alt: 'Location'}}
+        text={{txt: 'Вся Беларусь'}}
+      />
     </div>
     <div className="right-part-of-header">
-      <button className="btn--add-product">
-        <img className="img--plus" src={plus} alt="Plus" />
-        <p className="btn--add-ad__text btn--add-product__text-full">Подать объявление</p>
-        <p className="btn--add-ad__text btn--add-product__text-reduction">Объявление</p>
-      </button>
-      <Button />
-
+      <Button
+        className="btn--add-product btn-type-primary"
+        img={{src: plus, className: 'img--plus', alt: 'Plus'}}
+        text={{className: 'btn--add-ad__text btn--add-product__text-full', txt: 'Подать объявление'}}
```

Comment:
I thought we agreed to move these hard coded classnames into Button component, didn't we?

Reply:
fixed